core/src/main/kotlin/org/evomaster/core/problem/rest/RestCallAction.kt

```diff
@@ -46,7 +47,7 @@ class RestCallAction(
 
     override fun copy(): Action {
         val p = parameters.asSequence().map(Param::copy).toMutableList()
-        return RestCallAction(id, verb, path, p, auth, saveLocation, locationId)
+        return RestCallAction(id, verb, path, p, auth, saveLocation, locationId, produces)
```

Comment:
if `produces` is mutable, then here we have a bug: a change in the copy would change the state of this object